internal/dump/dump.go

```diff
@@ -0,0 +1,40 @@
+package dump
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis/v8"
+	"go.riyazali.net/sqlite"
+)
+
+type dump struct {
+	rdb *redis.Client
+}
+
+func (f *dump) Args() int           { return -1 }
+func (f *dump) Deterministic() bool { return false }
+func (f *dump) Apply(ctx *sqlite.Context, values ...sqlite.Value) {
+	var (
+		key    string
+		result *redis.StringCmd
+		err    error
+	)
```

Comment:
let's declare these "inline" as they get used/initialized with the `:=` (walrus)